airbyte-ci/connectors/connector_ops/connector_ops/utils.py

```diff
@@ -555,6 +556,20 @@ def normalization_tag(self) -> Optional[str]:
     def is_using_poetry(self) -> bool:
         return Path(self.code_directory / "pyproject.toml").exists()
 
+    @property
+    def is_released(self) -> bool:
```

Comment:
Nit: can you add a comment line here? Pretty obvious, sure, but why use OSS catalog for example?

Reply:
Should we for example insist on running https checks when we're pushing a connector from being available on OSS only to being available in cloud? Should those be two separate release levels?

Reply:
> but why use OSS catalog for example?
The cloud registry is a subset of the oss registry. Some connector are not on cloud but I still want to know if they are released.